.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -43,6 +43,14 @@ assignees: ''
 
 <!-- A description of what you expected to happen. -->
 
+## Acceptance Criteria
+
+<!-- A list of one or more requirements that must be met in order to for -->
+<!-- this ticket to be considered “done.” Use checkboxes to mark each item -->
+<!-- as an actionable task. -->
+
+- [ ] tk…
```

Comment:
```suggestion
- [ ] TK
```

This is so stupid and nitpicky that I wouldn't dare have it hold up this ticket, but two things here 🤦 I'm sorry…
- Can it be one word to make it one key arrow tap shorter when selecting and replacing the "TK"? (Blame yourself for showing me this key command in the first place 😏)
- If you're doing that, then you might as well update it to be a weeeeee bit less ambiguous with a classic "TK" (all uppercase like heyday editorial redlining) 🤷 

_If_ you let me be that kind of asshole, then you'd update the other two templates, too.